### PR TITLE
Add JAVA_HOME to the PATH

### PR DESCRIPTION
**Pull request by kdaniels**
...to temporarily fix travis-ci/travis-cookbooks#875

### CODE REVIEW

File: lib/travis/build/script/shared/jvm.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@ def setup
           CLEANUPS.each do |find_arg|
             sh.raw "find #{find_arg[:directory]} -name #{find_arg[:glob]} -delete 2>/dev/null"
           end
+          sh.export 'PATH', '$JAVA_HOME:$PATH'
```
Comment (BanzaiMan) [inline lib/travis/build/script/shared/jvm.rb]: It is `$JAVA_HOME/bin`, not `$JAVA_HOME`, which should be added.

Comment (kdaniels) [inline lib/travis/build/script/shared/jvm.rb]: Ahhh, good catch! Will fix, thanks!

**Comment by emdantrim** [comment]
We were trying to test these changes in staging but ran into some issues getting a test job to actually run in staging. We'll come back to this next week.

**Comment by kdaniels** [comment]
Tested and seems to work on staging: https://staging.travis-ci.org/kdaniels/obstinate-otter/jobs/615393#L429

**Pull request merged**